docs/modules/security/pages/native-client-security.adoc

```diff
@@ -1,68 +1,9 @@
-= Client Security
-:description: To protect your members from a malicious client, you can allow them to identify clients and restrict their permissions to access either data in data structures or features such as user code deployment.
+= Native Client Security
```

Comment:
Nit - as before, define `native client`?

Reply:
This page is a leftover and should be removed completely as its content is now part of the `./docs/modules/clients/pages/java.adoc`.

Reply:
Is removing it in scope of this PR, or to be raised seperately?